docs/configuration/rack-parameters/aws/additional_node_groups_config.md

```diff
@@ -26,7 +26,10 @@ The `additional_node_groups_config` parameter takes a JSON array of node group c
 
 | Field | Required | Description | Default |
 |-------|----------|-------------|---------|
-| `type` | Yes | The EC2 instance type to use for the node group |  |
+| `type` | Yes (or set the `cpu` and `mem` fields) | The EC2 instance type to use for the node group |  |
+| `cpu` | Yes (or set the `type` field) | The minimum number of vCPUs required from the node |  |
+| `mem` | Yes (or set the `type` field) | The minimum mib of memory required from the node |  |
```

Comment:
Why are `cpu` and `memory` necessary here?
Wouldn't just `types` be enough?

Reply:
nano and micro won't work considering running all those daemonsets and other. we can set that as minimum, omit cpu and mem as option. instance requirements should only be considering the allowed types in that case based on that minimum 2 cpu and 2 gib mem.